.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,27 @@
+name: Continuous integration
+on: [push]
+jobs:
+  backend:
+    runs-on: ubuntu-latest
+    env:
+      TOX_PARALLEL_NO_SPINNER: 1
+    steps:
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.6.9
+    - name: Update pip
+      run: python -m pip install --upgrade pip
+    - name: Install tox
+      run: python -m pip install tox
+    - name: Checkout git repo
+      uses: actions/checkout@v2
+    - name: Cache the .tox dir
+      uses: actions/cache@v2
+      with:
+        path: .tox
+        key: ${{ runner.os }}-tox-${{ hashFiles('tox.ini', 'requirements*', 'setup.py', 'setup.cfg') }}
```

Comment:
I don't know how much it matters, but there is currently no `setup.cfg` file in this repo. It looks like only the h, bouncer and lms projects have that file.

Reply:
Yeah, it doesn't seem to break `hashFiles()` and it just allows the key to be the same across all our repos